_security-analytics/usage/rules.md

```diff
@@ -82,18 +161,25 @@ status: experimental
 ```
 {% include copy.html %}
 
-1. To begin, select the **Import rule** button in the upper-right corner of the page. The Import rule page opens.
-1. Either drag a YAML-formatted Sigma rule into the window or browse for the file by selecting the link and opening it. The Import a rule window opens and the rule definition fields are automatically populated in both the Visual Editor and YAML Editor.
+To assist in rule creation using the **YAML Editor**, you can refer to Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide) and use the descriptions of each field to learn more about defining the rule.
+
+
+### Importing rules
+
+Security Analytics also supports the import of Sigma rules in YAML format. In the **Detection rules** window, follow these steps to import a rule.
+
+1. To begin, select **Import detection rule** in the upper-right corner of the page. The **Import rule** page opens.
+1. Either drag a YAML-formatted Sigma rule into the window or browse for the file by selecting the link and opening it. The **Import a rule** window opens and the rule definition fields are automatically populated in both the Visual Editor and YAML Editor.
 1. Verify or modify the information in the fields.
-1. After you confirm the information for the rule is accurate, select the **Create** button in the lower-right corner of the window. A new rule is created, and it appears in the list of rules on the main page of the Rules window.
+1. After you confirm that the information for the rule is accurate, select **Create detection rule** in the lower-right corner of the window. A new rule is created and it appears in the list of detection rules.
 
-## Customizing rules
+### Customizing rules
 
-An alternative to importing a rule is duplicating a Sigma rule and then modifying it to create a custom rule. First search for or filter rules in the Rules list to locate the rule you want to duplicate.
+Another option for creating a new detection rule is duplicating a Sigma rule and then modifying it to create a custom rule. First search for or filter rules in the **Rule name** list to locate the rule you want to duplicate.
 
 <img src="{{site.url}}{{site.baseurl}}/images/Security/rules-dup1.png" alt="Selecting a rule in the Rules name list" width="75%">
```

Comment:
It looks like this image needs to be referenced in the text that precedes it. Also, the list that follows should be introduced by a brief sentence ending in a colon.